lib/src/test/java/com/auth0/jwt/creators/AccessJwtCreatorTest.java

```diff
@@ -343,9 +343,9 @@ public void testAccessJwtCreatorExpTimeHasPassed() throws Exception {
     }
 
     private static void verifyClaims(Map<String,Claim> claims, Date exp) {
-        assertTrue(claims.get(PublicClaims.ISSUER).asList(String.class).get(0).equals("issuer"));
-        assertTrue(claims.get(PublicClaims.SUBJECT).asList(String.class).get(0).equals("subject"));
-        assertTrue(claims.get(PublicClaims.AUDIENCE).asString().equals("audience"));
-        assertTrue(claims.get(PublicClaims.EXPIRES_AT).asDate().toString().equals(exp.toString()));
+        assertTrue(claims.get(Claims.ISSUER).asString().equals("issuer"));
+        assertTrue(claims.get(Claims.SUBJECT).asString().equals("subject"));
+        assertTrue(claims.get(Claims.AUDIENCE).asString().equals("audience"));
+        assertTrue(claims.get(Claims.EXPIRES_AT).asDate().toString().equals(exp.toString()));
```

Comment:
is it better to compare the long timestamps or even just Date.equals?

Reply:
date.equals doesnt work.  i think this is the most accurate way

Reply:
what do you mean by "doesn't work"? If two classes are of the same type and have valid values I don't see why `equals` won't work.